selendroid-standalone/src/main/java/io/selendroid/standalone/SelendroidConfiguration.java

```diff
@@ -99,6 +99,10 @@
   @Parameter(names = "-maxInstances", description = "Maximum number of instances that a grid hub can use at a time.")
   private int maxInstances = 5;
 
+  @Parameter(names = "-registerCycle", description = "How often in ms the node will try to register itself again" +
```

Comment:
Works for me. I'd add it to the documentation though (`0 to disable`). Should `0` also be the default value?